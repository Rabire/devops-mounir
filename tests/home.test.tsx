import { render, screen } from "@testing-library/react";
import Home from "../app/page";

describe("Home", () => {
  it("renders a link", () => {
    render(<Home />);

    const heading = screen.getByRole("link", {
      name: /En savoir plus !/,
    });
    expect(heading).toMatchSnapshot();
  });
});
