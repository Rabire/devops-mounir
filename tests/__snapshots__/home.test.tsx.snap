// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders a link 1`] = `
<a
  class="bg-purple-600 text-white p-4 rounded font-medium"
  href="/count"
>
  En savoir plus !
</a>
`;
